pkg/sfu/receiver.go

```diff
@@ -470,6 +471,14 @@ func (w *WebRTCReceiver) SendRTCP(p []rtcp.Packet) {
 	w.rtcpCh <- p
 }
 
+func (w *WebRTCReceiver) SendPLI(layer int32) {
+	pli := []rtcp.Packet{
+		&rtcp.PictureLossIndication{SenderSSRC: rand.Uint32(), MediaSSRC: w.SSRC(int(layer))},
+	}
+
+	w.SendRTCP(pli)
```

Comment:
I believe PLI is the only kind of message that downtracks send up to receiver. Perhaps in a diff PR, it'd be good to update that interface to only expose `SendPLI(layer)`.

Reply:
Will look at that @davidzhao . Currently, is forwarding packets as is from `handleRTCP` too which includes FIR in addition to PLI. Yeah, worth cleaning it up some.